python/fatcat_web/routes.py

```diff
@@ -1283,14 +1283,14 @@ def stats_json() -> AnyResponse:
 @crossdomain(origin="*", headers=["access-control-allow-origin", "Content-Type"])
 def container_issnl_stats(issnl: str) -> AnyResponse:
     if not (len(issnl) == 9 and issnl[4] == "-"):
-        abort(400, "Not a valid ISSN-L: {}".format(issnl))
+        abort(400, f"Not a valid ISSN-L: {issnl}")
     try:
         container = api.lookup_container(issnl=issnl)
     except ApiException as ae:
         raise ae
     try:
         stats = get_elastic_container_stats(container.ident, issnl=container.issnl)
-    except (ValueError, IOError) as ae:
```

Comment:
Huh, today I learned, IOError and OSError are merged: https://peps.python.org/pep-3151/#step-1-coalesce-exception-types